Box.V2/Utility/ExponentialBackoff.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Do you need all of these?  It doesn't look like you use many different classes below.